src/features/classification/applied-by-ai-classification-reason/AppliedByAiClassificationReason.scss

```diff
@@ -0,0 +1,31 @@
+@import '~@box/blueprint-web-assets/dist/tokens/tokens';
```

Comment:
try to use `var(...)` variables instead of importing tokens. this allows for branding and theming to overrides values